src/docs/reference/modules/Contents/README.md

```diff
@@ -129,6 +129,125 @@ Well known properties include
+
+### Activating the "OpenId Authorization Server" and "OpenId Token Validation" Features, and setting User Roles
+
+To utilize the Orchard Core ApiController endpoints, user accounts must authenticate using the OAuth 2 standard. This requires activating the "OpenId Authorization Server" feature and configuring it (detailed information can be found in the [documentation](https://docs.orchardcore.net/en/main/docs/reference/modules/OpenId/#authorization-server)). Furthermore, the "OpenId Token Validation" feature needs to be activated, which provides a token to authenticated users for future API calls (more details are available in the [documentation](https://docs.orchardcore.net/en/main/docs/reference/modules/OpenId/#token-validation)). Upon activating these features, the administration area's "Security" section will display a new menu option "OpenID Connect," enabling the OAuth 2 server configuration through the creation of "Scopes" and "Applications".
+
+It is usually better to create a dedicated user for performing API calls, to maintain control over user rights, and to easily activate/deactivate the API user. The `OrchardCore.OpenId` feature allows setting these user role permissions from "Roles > Edit (User)":
+
+- View and edit the OpenID Connect client settings
+- View and edit the OpenID Connect server settings
+- View and edit the OpenID Connect validation settings
+- View, add, edit, and remove the OpenID Connect applications
+- View, add, edit, and remove the OpenID Connect scopes.
+
+### Contents API Controller Endpoints
+
+#### <span style="color:blue">GET /api/content/{contentItemId} </span>
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| contentItemId | path |  | Yes | string |
+
+##### Responses
+
+| Code | Description |
+| ---- | ----------- |
+| 200 | Success |
+
+***
+
+#### <span style="color:blue">GET /api/queries/{name} </span>
+##### Parameters
+
+!!! note
+    This endpoint can only be used when the GraphQL feature is active
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| name | path |  | Yes | string |
+| parameters | query |  | No | string |
+
+##### Responses
+
+| Code | Description |
+| ---- | ----------- |
+| 200 | Success |
+
+***
+
+#### <span style="color:green">POST /api/content </span>
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| draft | query |  | No | boolean |
+
+##### Body payload example
```

Comment:
Add notes how one can create a payload themselves for any content item (like exporting an item and using that JSON as a start).